.github/workflows/tests.yml

```diff
@@ -33,7 +33,8 @@ jobs:
               uses: shivammathur/setup-php@2.18.1
               with:
                   php-version: ${{ matrix.php-versions }}
-                  extensions: gd,mbstring,pcov,xdebug
+                  coverage: xdebug
```

Comment:
just a qustion: is there any benefit of using `xdebug` for **only** coverage when you don't use branch coverage? IMO `pcov` is much faster

Reply:
you're right, I found a way to get it working !